### DISCUSSION TITLE
how to install libraries in windows

### DISCUSSION THREAD

**Discussion by sarathbabuSwamynathan** [tjko/jpegoptim#89] [General]
I have successfully installed all libraries in linux server. Don't know how to configure with windows server. help me to proceed further.

**Comment by dreamlayers** [comment]
First, build mozjpeg. Then, configure jpegoptim with `CPPFLAGS=-I/path/to/mozjpeg` and `LDFLAGS=-L/path/to/mozjpeg/.libs`. Now, you could put the mozjpeg DLL somewhere in your PATH or in the same directory as jpegoptim.exe, but I recommend instead manually re-doing the final link using `libjpeg.a` instead of `-ljpeg`. Then the executable will be statically linked and you won't have to worry about the DLL.

**Comment by XhmikosR** [comment]
Alternatively, see https://github.com/XhmikosR/jpegoptim-windows.

**Comment by Murmur** [comment]
Anyone still googling and found this topic. See this [comment in jpegoptim](https://github.com/tjko/jpegoptim/pull/115#issuecomment-2768339985) how to build a windows binary with `VisualStudio2022(community)` cmdline.
